grails-app/controllers/org/pih/warehouse/api/CycleCountApiController.groovy

```diff
@@ -65,4 +68,28 @@ class CycleCountApiController {
 
         render([data: cycleCount] as JSON)
     }
+
+    def updateCycleCountItem(CycleCountUpdateItemCommand command) {
+        if (command.hasErrors()) {
+            throw new ValidationException("Invalid cycle count item", command.errors)
+        }
+        CycleCountItemDto cycleCountItem = cycleCountService.updateCycleCountItem(command)
+
+        render([data: cycleCountItem] as JSON)
+    }
+
+    def deleteCycleCountItem() {
```

Comment:
Would be good to standardize on a convention where we bind the ID as a method argument or the resource we are deleting. Both of these should work. The question is whether we need or want to do the lookup here vs within the service method.
```
def deleteCycleCountItem(String cycleCountItemId) 
def deleteCycleCountItem(CycleCountItem cycleCountItem)
```

Reply:
yeah I like having the full CycleCountItem as a method arg. It's clearest to me. So if that's an option it gets my vote

Reply:
The:
```
def deleteCycleCountItem(CycleCountItem cycleCountItem)
```
doesn't work. It's probably due to the fact that the query params are not bound for `DELETE` action. 
As you @jmiranda recently discovered - this would only be bound if it was `POST` action.

The version with String works fine, so I will go with that.